data/deploy.sh.erb

```diff
@@ -36,6 +36,7 @@ build_path="./tmp/build-$(date +%s)$RANDOM"
 
 version="<%= next_version %>"
 release_path="<%= fetch(:releases_path) %>/$version"
+release_relative_path="releases/$version"
```

Comment:
Either we do all relative or all absolute.

Or maybe add a configuration variable which would set only the current as relative

Now there are shared paths that are absolute...